core/static/js/dit.class.modal.js

```diff
@@ -223,7 +242,6 @@
     var self = this;
     self.$content.find("video, a, button, input, select").eq(0).focus();
```

Comment:
Taking into account the above comment, the `Modal.prototype.focus` function should be updated to simply be:
`self.$firstFocusElement.focus();`